src/file/preader.cr

```diff
@@ -13,7 +13,7 @@ class File::PReader < IO
     @pos = 0
```

Comment:
Not directly related, but `pos` really should be `Int64`

Reply:
I added it anyways. It's in a similar mood.